debian/control

```diff
@@ -1,10 +1,14 @@
 Source: fossology
 Section: utils
-Priority: extra
+Priority: optional
 Maintainer: Michael Jaeger <michael.c.jaeger@siemens.com>
-Build-Depends: debhelper, libglib2.0-dev, libmagic-dev, libxml2-dev, libmxml-dev, libtext-template-perl, librpm-dev, subversion, rpm, libpcre3-dev, libssl-dev, postgresql-server-dev-all, libboost-regex-dev, libboost-program-options-dev, libjsoncpp-dev, libjson-c-dev, php5-cli|php7.0-cli|php7.2-cli, php-mbstring|php5, php-zip|php5, php-xml|php5
+Build-Depends: debhelper (>=9~), libglib2.0-dev, libmagic-dev, libxml2-dev,
+ libmxml-dev, libtext-template-perl, librpm-dev, subversion, rpm, libpcre3-dev,
+ libssl-dev, postgresql-server-dev-all, libboost-regex-dev,
+ libboost-program-options-dev, libjsoncpp-dev, libjson-c-dev,
+ php5-cli|php7.0-cli|php7.2-cli, php-mbstring|php5, php-zip|php5, php-xml|php5
 Standards-Version: 3.9.1
-Homepage: http://fossology.org
+Homepage: https://fossology.org
 
 Package: fossology-dev
 Architecture: any
```

Comment:
W: fossology source: debhelper-but-no-misc-depends fossology-dev
need to put misc dependecy

Reply:
Fixed, please check again.